raiden-network/light-client
# dApp: refactor channels list, introduce withdraw dialog

**Thank you for submitting this pull request :)**

Last part of #249 

**Short description**
Withdraw screen, refactored channels list:
![image](https://user-images.githubusercontent.com/587021/87211888-295c2400-c2f2-11ea-9763-55b8b1e4de68.png)


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Access the Channels list screen on dApp
2. Click on the Withdraw button on a channel with capacity with the hub
3. Accept transaction and see it reduce the capacity; after confirmation, tokens should be transferred to account
